docs/site/migration/auth/built-in.md

```diff
@@ -1,17 +1,290 @@
 ---
 lang: en
-title: 'Migrating built-in authentication and authorization'
+title: 'Migrating built-in authentication'
 keywords: LoopBack 4.0, LoopBack 4, LoopBack 3, Migration
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/migration-auth-built-in.html
+permalink: /doc/en/lb4/migration-authentication.html
```

Comment:
Do we need to modify the `lb4_sidebar.yml` file if the link is changed?

Reply:
yep good catch!